### DISCUSSION TITLE
How to configure doubled translation files on stop Weblate from touching <source> tags?

### DISCUSSION THREAD

**Discussion by FrostKiwi** [WeblateOrg/weblate#9981] [Q&A]
Hey there.
We have Angular 13 + [ng-extract-i18n-merge](https://github.com/daniel-sc/ng-extract-i18n-merge) providing the XLIFF files, that Weblate ingests. The source language is English. The issue is when translators find corrections to be done in the English version as well and correct it, then Weblate changes the Source strings as well. So the Weblate English -> English perform a modification of both `<source></source>` and `<target></target>`. This works fine when deploying, but confuses [ng-extract-i18n-merge](https://github.com/daniel-sc/ng-extract-i18n-merge), which led to issue: https://github.com/daniel-sc/ng-extract-i18n-merge/issues/75

To resolve the issue, the maintainer of [ng-extract-i18n-merge](https://github.com/daniel-sc/ng-extract-i18n-merge) suggested ( https://github.com/daniel-sc/ng-extract-i18n-merge/issues/75#issuecomment-1709819196 ) to create a separate translation file for English, in conjunction with a code modification in [ng-extract-i18n-merge](https://github.com/daniel-sc/ng-extract-i18n-merge).

Having done so, I have translation files consisting of:
```
messages.de.xlf
messages.en.xlf
messages.ja.xlf
messages.ru.xlf
messages.xlf
```
This leads to Weblate complaining with:
> ## Duplicated translation. 
> The component contains translation file for the source language. 
> Please consider the following:
> - [In case translation to the source language is desired, please change the source language in the component settings.](https://docs.weblate.org/en/weblate-4.18.2/admin/projects.html#component-source-language)
> - If the translation file for the source language is not needed, please remove it from the repository.
> - [If the translation file for the source language is needed, but should be ignored by Weblate, please adjust the language filter to exclude it.](https://docs.weblate.org/en/weblate-4.18.2/admin/projects.html#component-language-regex)
> 
> The following occurrences were found:
> Language | Language codes | File names
> -- | -- | --
> English | en, en | frontend/src/locale/messages.en.xlf, frontend/src/locale/messages.xlf

How do I configure Weblate to:
 - Ignore `messages.xlf`
 - Perform English -> English translations as `<target state="translated"></target>`
 - And most importantly: **Not** perform any modifications of `<source></source>` tags.

**Comment by nijel** [comment]
Also discussed at https://github.com/WeblateOrg/weblate/issues/10024
